### ISSUE TITLE
torch 1.8 cannot torch.jit.load for script model

### ISSUE THREAD

**Issue by loganxiayan** [pytorch/pytorch#116498]
### 🐛 Describe the bug

When I load the script file with torch.jit.load in torch1.8, I can load the file obtained by jit.trace but not the file obtained by jit.script. how can I solve this problem.

### Versions

torch 1.8

cc @EikanWang @jgong5 @wenzhe-nrv @sanchitintel

**Comment by albanD** [comment]
Hey!

TorchScript is mostly in maintenance mode.
Also I would recommend using the latest version available since 1.8 is very old at this point.

**Comment by loganxiayan** [comment]
> Hey!
> 
> TorchScript is mostly in maintenance mode. Also I would recommend using the latest version available since 1.8 is very old at this point.

Yes 1.8 is an old version but there is a difference between its exported onnx and the new version's exported onnx, so I had to consider using version 1.8

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
